src/pages/ProductDetail/Count.scss

```
@@ -0,0 +1,21 @@
+.count {
+  display: flex;
+  line-height: 16px;
+  .countBtn {
+    width: 20px;
```

```suggestion
  line-height: 16px;
  
  .countBtn {
    width: 20px;
```
선택자끼리 가독성을 위해 한칸 씩 띄워주세요!